CONTRIBUTING.md

```diff
@@ -53,15 +51,15 @@ about conventions and design patterns found in the project.
 
 If you want to write some code, but don't know where to start or what you might
 want to do, take a look at our [Unplanned][] milestone. This is where you can
-find issues we would like to address but can't currently find time for. See if
+find issues we would like to address, but can't currently find time for. See if
```

Comment:
We may also want to mention the [Good First Issue label](https://github.com/google/go-cloud/labels/good%20first%20issue).